### PR TITLE
DT-414: Add encoding function and use it

### PR DESCRIPTION
**Pull request by Janrain-Colin**

### CODE REVIEW

**Comment by cloeperJanrain** [comment]
Looks good!

**Comment by JanrainMicah** [comment]
Needs to be untabbed, tagged, and released to PyPi so it can be dog food?

**Comment by Janrain-Colin** [comment]
Exactly.

File: janrain/__init__.py [hunk 1]
```diff
@@ -1 +1,13 @@
 __import__('pkg_resources').declare_namespace(__name__)
+
+def utf8_encode(s):
+	try:
+		if isinstance(s, unicode):
```
Comment (aldenatjanrain) [inline janrain/__init__.py]: Did someone use tabs?

**Pull request merged**